hubstorage/client.py

```diff
@@ -40,7 +40,7 @@ def _hc_retry_on_exception(err):
 class HubstorageClient(object):
 
     DEFAULT_ENDPOINT = 'https://storage.scrapinghub.com/'
-    USERAGENT = 'python-hubstorage/{0}'.format(__version__)
+    DEFAULT_USER_AGENT = 'python-hubstorage/{0}'.format(__version__)
```

Comment:
Agree, it's better. We still don't raise a warning when somebody reads `cls.USERAGENT` (like we did it via property), but I guess it's an acceptable trade-off and it's fine to just ignore such cases for now.